plugins/slack_webhooks/app/views/samson_slack_webhooks/_notify_buddy_box.html.erb

```diff
@@ -0,0 +1,67 @@
+<% if @deploy.waiting_for_buddy? && @deploy.stage.send_slack_webhook_notifications? %>
```

Comment:
can we render a shared view here ?
move this into a app/view/shared/_buddy_request and reuse it